Gemfile

```diff
@@ -19,3 +19,8 @@ group :development, :test do
   gem 'pry'
   gem 'generator_spec'
 end
+
+group :test do
+  gem 'test-unit'
```

Comment:
@domaindrivendev when I was doing bundle exec rspec (without any changes), it was failing with error test-unit failing. 

```
bundler: failed to load command: rspec (/usr/local/var/rbenv/versions/2.3.0/bin/rspec)
LoadError: Please add test-unit gem to your Gemfile: `gem 'test-unit', '~> 3.0'` (cannot load such file -- test/unit/testcase)
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/activesupport-3.2.22/lib/active_support/test_case.rb:2:in `require'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/activesupport-3.2.22/lib/active_support/test_case.rb:2:in `<top (required)>'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/railties-3.2.22/lib/rails/generators/test_case.rb:30:in `require'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/railties-3.2.22/lib/rails/generators/test_case.rb:30:in `<module:Generators>'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/railties-3.2.22/lib/rails/generators/test_case.rb:9:in `<module:Rails>'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/railties-3.2.22/lib/rails/generators/test_case.rb:8:in `<top (required)>'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/generator_spec-0.9.3/lib/generator_spec/test_case.rb:2:in `require'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/generator_spec-0.9.3/lib/generator_spec/test_case.rb:2:in `<top (required)>'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/generator_spec-0.9.3/lib/generator_spec/generator_example_group.rb:1:in `require'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/generator_spec-0.9.3/lib/generator_spec/generator_example_group.rb:1:in `<top (required)>'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/generator_spec-0.9.3/lib/generator_spec.rb:2:in `require'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/generator_spec-0.9.3/lib/generator_spec.rb:2:in `<top (required)>'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.12.5/lib/bundler/runtime.rb:86:in `require'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.12.5/lib/bundler/runtime.rb:86:in `block (2 levels) in require'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.12.5/lib/bundler/runtime.rb:81:in `each'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.12.5/lib/bundler/runtime.rb:81:in `block in require'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.12.5/lib/bundler/runtime.rb:70:in `each'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.12.5/lib/bundler/runtime.rb:70:in `require'
  /usr/local/var/rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/bundler-1.12.5/lib/bundler.rb:102:in `require'
  /Users/<path-in-my-local-to-project>/swagger_rails/spec/dummy/config/application.rb:11:in `<top (required)>'
  /Users/<path-in-my-local-to-project>/swagger_rails/spec/dummy/config/environment.rb:2:in `require'
  /Users/<path-in-my-local-to-project>/swagger_rails/spec/dummy/config/environment.rb:2:in `<top (required)>'
  /Users/<path-in-my-local-to-project>/swagger_rails/spec/rails_helper.rb:3:in `require'
  /Users/<path-in-my-local-to-project>/swagger_rails/spec/rails_helper.rb:3:in `<top (required)>'
  /Users/<path-in-my-local-to-project>/swagger_rails/spec/swagger_helper.rb:1:in `require'
  /Users/<path-in-my-local-to-project>/swagger_rails/spec/swagger_helper.rb:1:in `<top (required)>'
  /Users/<path-in-my-local-to-project>/swagger_rails/spec/dummy/spec/integration/blogs_spec.rb:1:in `require'
  /Users/<path-in-my-local-to-project>/swagger_rails/spec/dummy/spec/integration/blogs_spec.rb:1:in `<top (required)>'
```